core/templates/domain/story/story-update.service.ts

```diff
@@ -35,11 +35,19 @@ import { StoryNode } from './story-node.model';
 
 type StoryUpdateApply = (storyChange: StoryChange, story: Story) => void;
 type StoryUpdateReverse = (storyChange: StoryChange, story: Story) => void;
-
+type Apply = (
+  backendChangeObject: BackendChangeObject,
+  domainObject: DomainObject) => void;
+type Reverse = (
+  backendChangeObject: BackendChangeObject,
+  domainObject: DomainObject) => void;
+
+// For properties like initialNodeId, thumbnailBackdroundColor
+// old value can be null.
```

Comment:
Please move this above the old_value filed directly.

Reply:
Done.